lua/bible-verse/commands.lua

```diff
@@ -39,7 +46,7 @@ function M.setup()
 		local command = vim.trim(args.args or "")
 		M.cmd(command)
 	end, {
-		nargs = "?",
+		nargs = "*",
```

Comment:
Doesn't seem that any of the new commands need to accept >1 arguments 🤔

Reply:
Indeed, I forgot to change it back